app/store/saga/index.js

```diff
@@ -20,4 +20,4 @@ export default function* rootSaga() {
     ...watchDownloadedContent
   ])
 }
-//all saga put here its a root of all saga 
+//all saga put here its a root of all saga
```

Comment:
remove comment if not needed

Reply:
this is needed as a kind of information.